File: tools/mo/openvino/tools/mo/front/caffe/tanh.py
# Copyright (C) 2018-2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0

from openvino.tools.mo.ops.activation_ops import Tanh
from openvino.tools.mo.front.extractor import FrontExtractorOp


class TanhFrontExtractor(FrontExtractorOp):
    op = 'Tanh'
    enabled = True

    @classmethod
    def extract(cls, node):
        Tanh.update_node_stat(node)
        return cls.enabled

File: src/core/tests/type_prop/tanh.cpp
// Copyright (C) 2018-2025 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//

#include "openvino/op/tanh.hpp"

#include "unary_ops.hpp"

using Type = ::testing::Types<ov::op::v0::Tanh>;

INSTANTIATE_TYPED_TEST_SUITE_P(type_prop_tanh, UnaryOperator, Type);
